Makefile

```diff
@@ -86,7 +85,8 @@ fmt:
 
 lint:
 	@echo "+ $@"
-	@test -z "$$(golint ./... | grep -v .pb. | grep -v Godeps/_workspace/src/ | tee /dev/stderr)"
+	# TODO: remove trustmanager/yubikey/ from ignored once linted
```

Comment:
It's this https://github.com/runcom/notary/blob/invert-pkcs11-buildtag/Makefile#L21